app/test_main.py

```diff
@@ -2,2 +1,47 @@
+import pytest
 
-# write your code here
+from .main import get_human_age
+
+
+def test_add_not_int_argument() -> None:
+    with pytest.raises(TypeError):
+        get_human_age(12.52, "52.12")
+
+
+def test_negative_value() -> None:
+    assert (get_human_age(-12, -2) == [0, 0]), \
+        "The age shouldn't be negative"
+
+
+def test_get_human_age_should_return_list_two_elem() -> None:
+    assert isinstance(get_human_age(16, 15), list)
```

Comment:
The test `test_get_human_age_should_return_list_two_elem` checks if the result is a list, which is a good validation step. Consider also checking the length of the list to ensure it contains exactly two elements.